cmd/sigclient.go

```diff
@@ -51,6 +51,38 @@ var esBulkCmd = &cobra.Command{
 	},
 }
 
+// esBulkCmd represents the es bulk ingestion
+var createLogsCmd = &cobra.Command{
+	Use:   "create-logs",
+	Short: "ingest records to SigScalr using es bulk",
+	Run: func(cmd *cobra.Command, args []string) {
+		processCount, _ := cmd.Flags().GetInt("processCount")
+		dest, _ := cmd.Flags().GetString("dest")
+		totalEvents, _ := cmd.Flags().GetInt("totalEvents")
+		continuous, _ := cmd.Flags().GetBool("continuous")
+		batchSize, _ := cmd.Flags().GetInt("batchSize")
+		indexPrefix, _ := cmd.Flags().GetString("indexPrefix")
+		numIndices, _ := cmd.Flags().GetInt("numIndices")
+		generatorType, _ := cmd.Flags().GetString("generator")
+		ts, _ := cmd.Flags().GetBool("timestamp")
+		dataFile, _ := cmd.Flags().GetString("filePath")
+		indexName, _ := cmd.Flags().GetString("indexName")
+		bearerToken, _ := cmd.Flags().GetString("bearerToken")
+
+		log.Infof("processCount : %+v\n", processCount)
+		log.Infof("dest : %+v\n", dest)
+		log.Infof("totalEvents : %+v. Continuous: %+v\n", totalEvents, continuous)
+		log.Infof("batchSize : %+v\n", batchSize)
+		log.Infof("indexPrefix : %+v\n", indexPrefix)
+		log.Infof("indexName : %+v\n", indexName)
+		log.Infof("numIndices : %+v\n", numIndices)
+		log.Infof("bearerToken : %+v\n", bearerToken)
+		log.Infof("generatorType : %+v. Add timestamp: %+v\n", generatorType, ts)
+
+		ingest.StartIngestion(ingest.CreateLogs, generatorType, dataFile, totalEvents, continuous, batchSize, dest, indexPrefix, indexName, numIndices, processCount, ts, 0, bearerToken)
```

Comment:
this is not needed, we are just adding a new reader type, and the whole flow should just work, we don't need any new `case ` statement either